_pages/security/incident-response-drills.md

```diff
@@ -0,0 +1,228 @@
+
+
+### Example drill steps:
+
+1. Acknolwedge that the first step would be to inform points of contact; establish that everyone knows who to inform in the event of an incident.
+1. Choose a scenario to drill: compromised GitHub account, compromised cloud.gov account, or compromised deploy key. (Compromised deploy key might be easiest to drill)
+1. Practice the steps to remove compromised accounts or credentials, for example, by deleting the current deployment keys, remaking them, and adding them to CI/CD.
+1. Using a development application instance, practice removing the route to a instance that may have been compromised and then renaming it to preserve forensics.
+
+## Scenario: PII Exposed
+
+It's discovered that PII is being leaked to unauthorized users through the site.
+
+### Example mitigation steps:
+
+1. Contact `<<Insert email of POC laid out in Agency policies>>` and inform them of a breach.
+1. Stop the exposure.
+  * Assess the severity and impact of the potential leak.
+  * Decide if the site needs to be set into a maintenance mode to stop further exposure. If yes, then bring up the maintenance page.
+  * If you are able to isolate the section of the site where the issue is occurring and remove/hide the page.
+1. Identify root cause of the issue and deploy a hotfix.
+1. Take necessary corrective action as directed by your agency security team. If there are corrective actions that the PO is able to handle in terms of contacting the affected users, do so.
+
+### Example drill steps:
+
+1. Acknolwedge that the first step would be to inform points of contact; establish that everyone knows who to inform in the event of an incident.
+1. In a development environment, practice putting the site into a maintenance mode or removing/hiding a page on the site, whichever would be most relevant to your project.
+1. Review any relevant corrective action / affected user notification procedures.
+
+
+## Scenario: Oops, I Deleted the Database
+
+The database needs to be restored from a backup.
+
+### Example mitigation steps:
+
+1. If you're using cloud.gov, follow [cloud.gov database backup procedures](https://cloud.gov/docs/services/relational-database/#backups).
+
+### Example drill steps:
+
+Assuming you have a staging database using a dedicated cloud.gov database plan:
+
+1. Delete some data from your staging database. (No deleting data from a production database, please.)
+2. Reach out to cloud.gov using the [the non-emergency email address provided in thir docs](https://cloud.gov/docs/services/relational-database/#backups); request a backup.
+3. Practice restoring the staging database to the point in time before you deleted the data.
+
+## Scenario: Oops, I Erased the S3 Bucket
+
+Let's re-create and restore from a backup.
+
+### Example mitigation steps:
+
+1. If the bucket no longer exists, create a new bucket in cloud.gov in the space where the bucket was deleted, ideally using infrastructure-as-code or a deploy script.
```

Comment:
we don't have a handbook page on infrastructure as code, do we? maybe we should. (out of scope for this, just thinking aloud on your PR!)

Reply:
I don't think we do, and that would be a super helpful resource ... the Tech Lead I'm working with has asked really great questions about infrastructure as code and how to know what good IaC practices look like.